README.md

```diff
@@ -547,6 +547,40 @@ Restore (re-reference) the hotplug events unreferenced by `unrefHotplugEvents()`
 #### usb.unrefHotplugEvents();
 Listening to events will prevent the process to exit. By calling this function, hotplug events will be unreferenced by the event loop, allowing the process to exit even when listening for the `attach` and `detach` events.
 
+# Migrating from node-usb-detection
+
+If you have be referred here by node-usb-detection, the following may be helpful for you to update your existing code.
+
+## usbDetect.startMonitoring() & usbDetect.stopMonitoring()
+
+There is no direct equivalent to these methods. This is handled automatically for you when you add and remove event listeners.
+
+You may find `usb.unrefHotplugEvents()` useful as it is intended to help with exit conditions.
+
+## usbDetect.find()
+
+You can instead use `usb.getDeviceList()`. Be aware that this will do an enumeration to find all of the devices, and not look at a cache of the known devices. If you call it too often it may have a performance impact.
+
+The api does not allow for filtering by vid and pid, but you can do this yourself by doing a `usb.getDeviceList().filter((dev) => dev.deviceDescriptor.idVendor === 0x12 && dev.deviceDescriptor.idProduct === 0x34)`
```

Comment:
This is incorrect, you can use `findByIds` https://github.com/node-usb/node-usb#find-legacy-device-by-vidpid